### ISSUE TITLE
IDE integration problems

### ISSUE THREAD

**Issue by nikoncode** [google/protobuf-gradle-plugin#512]
Hi. I am using protobuf plugin (v 0.8.16 / gradle version 6.9) and compilation itself working fine, but I have IntellijIdea integration problems

1) IDE wasn't able to find generated classes. They are all red.  Proto sources located in default directory `src/main/proto` and generated sources are located at `build/generated/source/proto/main` and according to documentation should be attached automatically.
>This plugin integrates with the idea plugin and automatically registers the proto files and generated Java code as sources.
2) I connected `protoc-gen-validate` extension and plugin successfully extracted include protos to `build/extracted-include-protos/main/validate/validate.proto` but IDE highlight `import "validate/validate.proto";` red and wasn't able to locate it.

Content of my build gradle:
```
plugins {
	id 'org.springframework.boot' version '2.3.12.RELEASE'
	id 'io.spring.dependency-management' version '1.0.11.RELEASE'
	id 'com.google.protobuf' version '0.8.16'
	id 'java'
}

group = 'com.example'
version = '0.0.1-SNAPSHOT'
sourceCompatibility = '1.8'

repositories {
	mavenCentral()
	maven {
		url "https://packages.confluent.io/maven/"
	}
}

protobuf {
	plugins {
		javapgv {
			artifact = "io.envoyproxy.protoc-gen-validate:protoc-gen-validate:0.4.1"
		}
	}

	generateProtoTasks {
		all()*.plugins {
			javapgv {
				option "lang=java"
			}
		}
	}
}

dependencies {
	implementation 'io.confluent:kafka-protobuf-serializer:6.2.0'
	implementation 'org.springframework.boot:spring-boot-starter-validation'
	implementation 'org.springframework.boot:spring-boot-starter'
	implementation 'org.springframework.kafka:spring-kafka'
	implementation 'org.springframework.boot:spring-boot-starter-aop'
	implementation 'io.envoyproxy.protoc-gen-validate:pgv-java-stub:0.4.1'
	compileOnly 'org.projectlombok:lombok'
	annotationProcessor 'org.projectlombok:lombok'
	testImplementation('org.springframework.boot:spring-boot-starter-test') {
		exclude group: 'org.junit.vintage', module: 'junit-vintage-engine'
	}
	testImplementation 'org.springframework.kafka:spring-kafka-test'
}

test {
	useJUnitPlatform()
}
```

**Comment by nikoncode** [comment]
Everything was solved by adding `idea` plugins w/o any configuration. Probably this plugin depends optionally on idea plugin.

**Comment by voidzcy** [comment]
The issue should have been fixed by #506 but not released yet.
